request-handler/src/main/java/au/org/aodn/aws/wps/RequestHandler.java

```diff
@@ -8,19 +8,24 @@
 import javax.xml.bind.JAXBException;
 import javax.xml.bind.Marshaller;
 import java.io.StringWriter;
+import java.util.Properties;
 
 public class RequestHandler {
-    public AwsApiResponse handleRequest(AwsApiRequest request) {
+    public AwsApiResponse handleRequest(AwsApiRequest request, Properties config) {
         ResponseBuilder responseBuilder = new ResponseBuilder();
 
         try {
             JAXBContext context = JAXBContext.newInstance(ExecuteResponse.class);
             RequestParserFactory requestParserFactory = new RequestParserFactory(context);
             RequestParser requestParser = requestParserFactory.getRequestParser(request);
             Operation operation = requestParser.getOperation();
-            Object result = operation.execute();
+            System.out.println("Operation : " + operation.getClass());
```

Comment:
We can - i just noticed that System.out gets redirected to the CloudWatch logs for free - so used it for debugging in this prototype code.  We can use Log4J if we want - there is an appender you can add that directs output to the CloudWatch logs.  Or: we can use LambdaLogger.